### PR TITLE
Added test to multi-layer-selection component

### PR DESCRIPTION
**Pull request by KaroFr**

### CODE REVIEW

**Comment by KaroFr** [comment]
[Multi-Selection-Layer Componente testen](https://trello.com/c/AKrvekxZ/46-multi-selection-layer-componente-testen)

File: projects/wave-core-new/src/lib/operators/dialogs/helpers/multi-layer-selection/multi-layer-selection.component.spec.ts [hunk 1]
```diff
@@ -0,0 +1,208 @@
...
+    it('should display "No Input Available", if no Layer is available', () => {
+        html = fixture.nativeElement.querySelector('p');
+        expect(component.selectedLayers.value).toEqual([]);
+        expect(html.textContent).toEqual('No Input Available');
+    });
+
+    /** Adding three layers
+     * checking the number of possible layers to select from
+     * checking the default layer displayed **/
+    it('testing with min = max = 1', async () => {
```
Comment (jdroenner) [inline projects/wave-core-new/src/lib/operators/dialogs/helpers/multi-layer-selection/multi-layer-selection.component.spec.ts:120]: i guess we need a name like "should do something?"

File: projects/wave-core-new/src/lib/operators/dialogs/helpers/multi-layer-selection/multi-layer-selection.component.spec.ts [hunk 2]
```diff
@@ -0,0 +1,208 @@
...
+                expect(html.textContent).toEqual(component.selectedLayers.value[0].name);
+                expect(html.textContent).toEqual(mockLayers[i - 1].name);
+            });
+        }
+    });
+
+    /** Adding three layers
+     * adding two more input fields
+     * checking the default layers displayed and the number of possible layers to select **/
+    it('testing with max = 3', async () => {
```
Comment (jdroenner) [inline projects/wave-core-new/src/lib/operators/dialogs/helpers/multi-layer-selection/multi-layer-selection.component.spec.ts:155]: same as above

File: projects/wave-core-new/src/lib/operators/dialogs/helpers/multi-layer-selection/multi-layer-selection.component.spec.ts [hunk 3]
```diff
@@ -0,0 +1,208 @@
...
+        component.ngOnChanges({max: new SimpleChange(component.max, 3, true), layers: new SimpleChange(undefined, component.layers, true)});
+        fixture.detectChanges();
+        component.add();
+        component.add();
+        const amount = component.selectedLayers.value.length;
```
Comment (jdroenner) [inline projects/wave-core-new/src/lib/operators/dialogs/helpers/multi-layer-selection/multi-layer-selection.component.spec.ts:160]: can we assert the number?

File: projects/wave-core-new/src/lib/operators/dialogs/helpers/multi-layer-selection/multi-layer-selection.component.spec.ts [hunk 4]
```diff
@@ -0,0 +1,208 @@
...
+                    expect(html.textContent).toEqual(component.selectedLayers.value[j].name);
+                });
+            }
+        }
+
+        component.ngOnChanges({max: new SimpleChange(component.max, 3, true), layers: new SimpleChange(undefined, component.layers, true)});
+        fixture.detectChanges();
+        component.add();
+        component.add();
+        const amount = component.selectedLayers.value.length;
```
Comment (jdroenner) [inline projects/wave-core-new/src/lib/operators/dialogs/helpers/multi-layer-selection/multi-layer-selection.component.spec.ts:194]: same as above

**Review by jdroenner — CHANGES_REQUESTED** [review]
looks good to me. please look into the two questions and merge the main branch again.

File: projects/wave-core-new/src/lib/operators/dialogs/helpers/multi-layer-selection/multi-layer-selection.component.spec.ts (hunk 2 above)
Comment (KaroFr) [reply projects/wave-core-new/src/lib/operators/dialogs/helpers/multi-layer-selection/multi-layer-selection.component.spec.ts]: eb7fda1155207b8594d05d8c3a338321cca92556

File: projects/wave-core-new/src/lib/operators/dialogs/helpers/multi-layer-selection/multi-layer-selection.component.spec.ts (hunk 3 above)
Comment (KaroFr) [reply projects/wave-core-new/src/lib/operators/dialogs/helpers/multi-layer-selection/multi-layer-selection.component.spec.ts:160]: eb7fda1155207b8594d05d8c3a338321cca92556

File: projects/wave-core-new/src/lib/operators/dialogs/helpers/multi-layer-selection/multi-layer-selection.component.spec.ts (hunk 4 above)
Comment (KaroFr) [reply projects/wave-core-new/src/lib/operators/dialogs/helpers/multi-layer-selection/multi-layer-selection.component.spec.ts:195]: eb7fda1155207b8594d05d8c3a338321cca92556

File: projects/wave-core-new/src/lib/operators/dialogs/helpers/multi-layer-selection/multi-layer-selection.component.spec.ts (hunk 1 above)
Comment (KaroFr) [reply projects/wave-core-new/src/lib/operators/dialogs/helpers/multi-layer-selection/multi-layer-selection.component.spec.ts]: 83f8e2db9c0ad618c651dd501506460b412615d1

File: projects/wave-core-new/src/lib/operators/dialogs/helpers/multi-layer-selection/multi-layer-selection.component.spec.ts (hunk 2 above)
Comment (KaroFr) [reply projects/wave-core-new/src/lib/operators/dialogs/helpers/multi-layer-selection/multi-layer-selection.component.spec.ts]: 83f8e2db9c0ad618c651dd501506460b412615d1

**Pull request merged**